src/tls13.c

```diff
@@ -1036,6 +1036,69 @@ int DeriveMasterSecret(WOLFSSL* ssl)
 #define RESUMPTION_LABEL_SZ         10
 /* Resumption label for generating PSK associated with the ticket. */
 static const byte resumptionLabel[RESUMPTION_LABEL_SZ+1] = "resumption";
+
+static int DeriveResumptionPSKExpand(byte* okm, word32 okmLen, const byte* prk,
```

Comment:
Don't copy the code, increase size of constant MAX_TLS13_HKDF_LABEL_SZ.
May have FIPS implications. Not sure which API we validated.

Reply:
AFAIU kdf.h and kdf.c can't be changed anymore in this FIPS version. 
Also, the maximum ticketNonce size may be 255 bytes, may this be a problem for a stack variable?

Reply:
Usually the limit for the stack that we follow is 100 bytes.

Reply:
I'm more worried that we now have an implementation outside the boundary!
Since we can't change the code then a copy that allocates will have to do.
When compiling for small stack, 100 bytes total for a function.

Reply:
I added a new function that allocates and I guarded out the features from the current fips versions